src/arkode/arkode_mristep.c

```diff
@@ -93,16 +93,14 @@ void* MRIStepCreate(ARKRhsFn fse, ARKRhsFn fsi, sunrealtype t0, N_Vector y0,
   }
 
   /* Allocate ARKodeMRIStepMem structure, and initialize to zero */
-  step_mem = NULL;
-  step_mem = (ARKodeMRIStepMem)malloc(sizeof(struct ARKodeMRIStepMemRec));
+  step_mem = (ARKodeMRIStepMem)calloc(1, sizeof(*step_mem));
```

Comment:
Related to the discussion in #451 we should settle on the a style for `malloc`/`calloc` calls i.e., `sizeof(struct ARKodeMRIStepMemRec)` vs `sizeof(*step_mem)`.